label_studio/data_manager/serializers.py

```diff
@@ -222,7 +270,7 @@ class DataManagerTaskSerializer(TaskSerializer):
     predictions_model_versions = serializers.SerializerMethodField(required=False)
     avg_lead_time = serializers.FloatField(required=False)
     draft_exists = serializers.BooleanField(required=False)
-    updated_by = serializers.SerializerMethodField(required=False, read_only=True)
+    updated_by = UpdatedByDMField(required=False, read_only=True)
```

Comment:
the same as get_drafts()

Reply:
Are you sure the problem does appear for this field as well? As it is just as shallow override over `serializers.SerializerMethodField`